backend/src/main/kotlin/com/dinnye/backend/controller/TestController.kt

```diff
@@ -0,0 +1,19 @@
+package com.dinnye.backend.controller
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.core.env.Environment
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class TestController(
+    @Autowired
```

Comment:
Ezzel nem tudom, hogy akarunk-e törődni de az Autowired elvileg nem Meta már nagyon.

Source: [](https://www.linkedin.com/pulse/you-should-stop-using-spring-autowired-felix-coutinho/)

Reply:
True csak propertynek írtam be először és bent maradt.